### ISSUE TITLE
Regex domain rule failed to load and operate properly

### ISSUE THREAD

**Issue by liny01-nbsa** [aloneguid/bt#67]
Browser Tamer 3.6.2

The below configure to handle a local application like `http://localhost:51139/` that the port number changes but starts with `5`.

The rules are setup as

```
[browser:chrome:Default]
name = Yuan
arg = "%url%" "--profile-directory=Default"
user_arg = "--enable-features=IntelVpSuperResolution"
icon = C:\Users\liny01\AppData\Local\Google\Chrome\User Data\Default\Google Profile Picture.png
rule = scope:domain|type:regex|localhost:5....
rule = scope:domain|intel.com
rule = scope:domain|hemingwayapp.com
```

It works properly when it's just set

![image](https://github.com/aloneguid/bt/assets/38985073/06f694ae-538a-4207-b40d-fb0c1350f7d1)

But after exit BT and start it again, this rule shows empty in the config box.

![image](https://github.com/aloneguid/bt/assets/38985073/9327e35f-c49a-48a8-92e4-1e9a7b98b446)

And it's loaded with the default browser.

**Comment by aloneguid** [comment]
Please try to put `user_arg` value in the UI instead:
![image](https://github.com/aloneguid/bt/assets/3155189/db60c581-b065-41af-a095-96d9e931de3e)

**Comment by liny01-nbsa** [comment]
@aloneguid I tried as you suggested, same issue.

 - The config works as when I configured in the UI
   - Tt's saved to configuration file.
 - But if I "exit" bt, and start it again, that particular rule is loaded back from config file as an empty rule, and not working.

![image](https://github.com/aloneguid/bt/assets/38985073/cc5d3705-104c-43db-9e7a-429163695137)

**Comment by aloneguid** [comment]
Thanks for detailed report, looking at it now.

**Comment by aloneguid** [comment]
I've found the issue, anything with `:` in rule text will clear it. The fix is coming.

**Comment by aloneguid** [comment]
@liny01-nbsa I'm taking you up on the offer :) Would be grateful if you can test it, download from https://github.com/aloneguid/bt/actions/runs/7714649868 (bin artifact). Seems to be OK on my side so far.

**Comment by liny01-nbsa** [comment]
@aloneguid , the new build works for me too.

**Comment by aloneguid** [comment]
this is now released in 3.7.0

**Comment by liny01-nbsa** [comment]
Confirm the release tag 3.7.0 works as expected.
